src/parser.test.ts

```diff
@@ -0,0 +1,264 @@
+import { RQLParseError } from './errors';
+import { inside, normalizeSyntax, parse, splitArguments, stringToValue, trim, walkQuery } from './parser';
```

Comment:
You include the `parse` function but don't test it.

Reply:
fixed